doc/source/rllib-algorithms.rst

```diff
@@ -364,3 +364,17 @@ Tuned examples: `CartPole-v0 <https://github.com/ray-project/ray/blob/master/rll
    :language: python
    :start-after: __sphinx_doc_begin__
    :end-before: __sphinx_doc_end__
+
+Alpha Zero (contrib/AZ)
```

Comment:
Since AZ is a common acronym (e.g., availability zone), consider `contrib/AlphaZero` for clarity.